### ISSUE TITLE
Add boolean property 'multiLanguage' to propertyClass

### ISSUE THREAD

**Issue by odungern** [GfSE/SpecIF-Schema#16]
Not all properties of dataType xs:string are multi-language, for example those of type dcterms:identifier. Even more, those _must not_ have multiple languages; Otherwise information is largely obscured.

A user interface must know whether a property may or may not have multiple languages, so the proposal is to add a native boolean attribute 'multiLanguage' to a propertyClass, similar to 'multiple'. This attribute is only meaningful for properties of dataType xs:string.

The values of a property are unchanged: In case a propertyClass specifies "multiple": false and "multiLanguage": false, the values are represented by two nested arrays with a single text value ... without language attribute:

```
{    
    .
    .
    "values": [[{ "text": "lorem ipsum"}]],
    .
}    
```

**Comment by odungern** [comment]
Has been proposed for SpecIF Schema v1.2. Any data-set according SpecIF Schema v1.1 is compatible.

**Comment by odungern** [comment]
'singleLanguage' (to reverse the default) implemented with schema 1.2.
